vms/platformvm/txs/executor/create_chain_test.go

```diff
@@ -87,7 +88,8 @@ func TestCreateChainTxWrongControlSig(t *testing.T) {
 	// Replace a valid signature with one from another key
 	sig, err := key.SignHash(hashing.ComputeHash256(tx.Unsigned.Bytes()))
 	require.NoError(err)
-	copy(tx.Creds[0].(*secp256k1fx.Credential).Sigs[0][:], sig)
+	sigPos := len(tx.Creds) - 1
```

Comment:
subnet creation fees are non zero, which implies that credentials changed position. Tx sign credential are always append last, so this change should make test less brittle